README.md

```diff
@@ -1,7 +1,7 @@
 # QCElemental
 
 [![Build Status](https://github.com/MolSSI/QCElemental/workflows/CI/badge.svg?branch=master)](https://github.com/MolSSI/QCElemental/actions?query=workflow%3ACI)
-[![codecov](https://codecov.io/gh/MolSSI/QCElemental/branch/master/graph/badge.svg)](https://codecov.io/gh/MolSSI/QCElemental)
+[![codecov](https://img.shields.io/codecov/c/github/MolSSI/QCElemental.svg?logo=Codecov&logoColor=white)](https://codecov.io/gh/MolSSI/QCElemental)
```

Comment:
Why did we change this to a non-native badge?

Reply:
I haven’t favored native over shieldsio badges, and the latter are more customizable. This fixes the hard to read dark pink on black umbrella (matches psi).

Reply:
Ok, would mind propagating this to the rest of the ecosystem?